network/p2p/unstaked_translator.go

```diff
@@ -20,6 +20,7 @@ import (
 // and any peer ID's which don't follow these conventions are considered invalid.
 type UnstakedNetworkIDTranslator struct{}
 
+// TODO Rename this one to NewPublicNetworkIDTranslator once observer changes are merged
```

Comment:
Can we just rename it now?

Reply:
Addressed and removed most of the references to unstaked in the affected areas: https://github.com/GetElastech/flow-go/commit/a33fc58d87da30525dbfc69bb1689ad931bb0636

Reply:
This one also addresses related functions not part of observer: https://github.com/GetElastech/flow-go/commit/fd244066aa205fab709701e58b52a5e2f7780888